### PR TITLE
apollo: clarify group and APIGroup

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

File: apis/query/v1alpha1/query_types.go [hunk 1]
```diff
@@ -136,8 +136,8 @@ type QueryCondition struct {
 
 // A QueryOwner specifies how to query by owner.
 type QueryOwner struct {
-	// name is the name of the owner to match.
-	Group string `json:"group,omitempty"`
+	// APIGroup is the name of the owner to match.
+	APIGroup string `json:"APIGroup,omitempty"`
```
Comment (adamwg) [inline apis/query/v1alpha1/query_types.go:140]: Should this be `apiGroup` in JSON?

File: apis/query/v1alpha1/query_types.go [hunk 2]
```diff
@@ -171,17 +171,22 @@ type QueryOrder struct {
 	// +kubebuilder:validation:Enum=Asc;Desc
 	Namespace Direction `json:"namespace,omitempty"`
 
+	// APIGroup specifies how to order by API group.
+	//
+	// +kubebuilder:validation:Enum=Asc;Desc
+	APIGroup Direction `json:"group,omitempty"`
```
Comment (adamwg) [inline apis/query/v1alpha1/query_types.go:177]: JSON name doesn't match.

**Review by adamwg — COMMENTED** [review]
I'll make the matching change in v1alpha2.

File: apis/query/v1alpha1/query_types.go (hunk 1 above)
Comment (sttts) [reply apis/query/v1alpha1/query_types.go:140]: formally in Kube it must be APIGroup.

File: apis/query/v1alpha1/query_types.go (hunk 2 above)
Comment (sttts) [reply apis/query/v1alpha1/query_types.go]: fixed

File: apis/query/v1alpha1/query_types.go (hunk 1 above)
Comment (sttts) [reply apis/query/v1alpha1/query_types.go:140]: Actually, no:
> Acronyms should similarly only be used when extremely commonly known. All letters in the acronym should have the same case, using the appropriate case for the situation. For example, at the beginning of a field name, the acronym should be all lowercase, such as "httpGet". Where used as a constant, all letters should be uppercase, such as "TCP" or "UDP".

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md

Comment (sttts) [reply apis/query/v1alpha1/query_types.go]: fixed

**Review by adamwg — APPROVED** [review]
:shipit:

**Pull request merged**